plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1175,6 +1175,7 @@ blockly.block.attack_action=Fight attacker mobs back, call for help %1 %2
 blockly.block.attack_entity=Attack (in sight only %1) (nearby only %2) entities of type %3 %4
 blockly.block.attack_fly=Do flight attacks in radius %2 with speed factor %1 (only for flying entities) %3
 blockly.block.attack_on_collide=Do melee/contact attack with speed factor %1 (chase after lost sight %2) %3
+blockly.block.attack_with_range=Do melee attack with range %2 and with speed factor %1 (chase after lost sight %3) %4
```

Comment:
`Do melee attack with range %2 speed factor %1 (chase after lost sight %3) %4`

is a more standard way we list parameters in AI blocks